source/common/upstream/cluster_manager_impl.cc

```diff
@@ -1024,9 +1024,9 @@ void ClusterManagerImpl::updateClusterCounts() {
   if (all_clusters_initialized && ads_mux_) {
     const auto type_url = Config::getTypeUrl<envoy::config::cluster::v3::Cluster>();
     if (last_recorded_warming_clusters_count_ == 0 && !warming_clusters_.empty()) {
+      resume_cds_.reset();
```

Comment:
Why is this added here? shouldn't the following line just replace the ptr?

Reply:
true, removed